src/index.js

```diff
@@ -6,11 +6,9 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
     <BrowserRouter>
```

Comment:
Oh nice! The `BrowserRouter` you're rendering inside of the App component is going to cause issues and you should need it if you're rendering one here.